Rakefile

```diff
@@ -23,4 +27,13 @@ GitHubChangelogGenerator::RakeTask.new :changelog do |config|
       "labels" => ["bugfix"],
     },
   }
-end
\ No newline at end of file
+end
+
+RSpec::Core::RakeTask.new(:spec) do |t|
+  t.exclude_pattern = 'spec/acceptance/**/*_spec.rb'
+end
+
+desc 'Run acceptance tests'
+task :acceptance do
```

Comment:
Wont this be cyclical?

if an acceptance rake task already exists we don't need to wrap it (I don't think)

Reply:
There isn't actually an acceptance rake task with this gem, but that's because it doesn't seem to include litmus. I'll add the usual acceptance test gem group to the Gemfile and that should work.

Reply:
There aren't any acceptance tests for this tool so we can probably remove everything relating to them for now.